src/string.cr

```diff
@@ -907,19 +909,25 @@ class String
     byte_slice start, bytesize - start
   end
 
+  # Returns the codepoint of `Char` at the given *index*, or raises `IndexError` if out of bounds.
+  #
+  # See also: `Char#ord`.
   def codepoint_at(index)
     char_at(index).ord
   end
 
+  # Returns the `UInt8` at the given *index*, or raises `IndexError` if out of bounds.
```

Comment:
This should describe the meaning of the return value, not its type. The type could be added as return type restriction.
```suggestion
  # Returns the byte at the given *index*.
  #
  # Raises `IndexError` if out of bounds. 
```
Ditto below.